docs/refmans/gui/viselements/generic/toggle.md_template

```diff
@@ -1,66 +1,128 @@
     <img src="../toggle-lov-l.png" class="visible-light" width="80%"/>
     <figcaption>Toggle control with three buttons</figcaption>
 </figure>
-In this example, the [*lov*](#p-lov) property of the control is a list of three string elements. If
-the user selects one of the elements, [*value*](#p-value) is updated accordingly to the selected
-string.
+In this example, the [*lov*](#p-lov) property contains a list of three string elements. When the
+user selects one of them, the [*value*](#p-value) property updates to reflect the selected string. 
 
-Note that if the variable bound to [*value*](#p-value) is a Boolean value, then the control displays
-a switch control:
+If the [*value*](#p-value) property is bound to a Boolean variable, the control automatically
+displays as a switch:  
 <figure class="tp-center">
     <img src="../toggle-switch-d.png" class="visible-dark"  width="80%"/>
     <img src="../toggle-switch-l.png" class="visible-light" width="80%"/>
     <figcaption>Toggle control bound to a Boolean variable</figcaption>
 </figure>
-The property [*value*](#p-value) reflects the control's status as a Boolean value.
+In this case, the [*value*](#p-value) property represents the control's state as a Boolean (True or
+False).
 
-You can also use an arbitrary type for all the items (see the [example](#use-arbitrary-objects)).
+Additionally, you can use custom object types for toggle items as explained in this
+[example](#picking-arbitrary-objects).
 
 # Usage
 
-## Display a list of string
+## Picking from a list of string {data-source="gui/examples/controls/toggle_simple.py"}
 
-You can create a list of toggle buttons from a series of strings:
+You can create a set of toggle buttons based on a list of strings:
 
 !!! taipy-element
     default={value}
     lov=Item 1;Item 2;Item 3
 
-## Deselected state
+Users can select one of the available options, and the selected value is stored in the variable
+bound to the [*value*](#p-value) property.  
 
-In a toggle control, all buttons might be unselected when [*allow_unselect*](#p-allow_unselect) is
-set to True.<br/>
-If the user deselects all the toggle items, [*value*](#p-value) is set to None.
+For example, if the following variable is defined:
+```python
+value = "Item 2"
+```
+
+The toggle control will be displayed as:
+<figure class="tp-center">
+    <img src="../toggle-simple-d.png" class="visible-dark"  width="80%"/>
+    <img src="../toggle-simple-l.png" class="visible-light" width="80%"/>
+    <figcaption>Toggle control with a few choices</figcaption>
+</figure>
+
+## Deselected state {data-source="gui/examples/controls/toggle_unselect.py"}
+
+By default, one option is always selected in a toggle control. However, setting
+[*allow_unselect*](#p-allow_unselect) to True allows users to deselect all options.<br/>
+To deselect an item, users can click on the currently selected button. If all items are deselected,
+the [*value*](#p-value) property is set to `None`.
```

Comment:
Unselected_value?

Reply:
I... don't think there's anything like this, @FredLL-Avaiga...
Please tell me how I'm wrong.

Reply:
I guess we removed it as non necessary